src/main/java/org/geysermc/discordbot/dump_issues/ProxyProtocolDumpIssueCheck.java

```diff
@@ -20,7 +20,9 @@
             warnings.add("- `enable-proxy-protocol` should ONLY be enabled if you run a reverse UDP proxy in front of Geyser.");
         }
         if (configRemote.getBoolean("use-proxy-protocol")) {
-            warnings.add("- `use-proxy-protocol` should ONLY be enabled if your server supports PROXY protocol (it probably doesn't), or you have the same option enabled in your BungeeCord/Velocity config (it is off by default).");
+            warnings.add("- `use-proxy-protocol` should ONLY be enabled if either of these apply:");
+            warnings.add("      1. Your server supports PROXY protocol (this has nothing to do with if you're using BungeeCord or Velocity).");
+            warnings.add("      2. You have the exact same option enabled in your BungeeCord/Velocity config (it is off by default).");
```

Comment:
I have a bad feeling this is going to display poorly, because the line's length will probably cause them to wrap

Reply:
May also cause issues if the list gets un-ordered. Maybe worth using `\n` instead of adding a new entry.